include/beast/http/impl/basic_parser_v1.ipp

```diff
@@ -913,9 +989,9 @@ write(boost::asio::const_buffer const& buffer, error_code& ec)
                 flags_ |= parse_flag::skipbody;
                 break;
             case body_what::pause:
-                s_ = s_headers_done;
                 return used();
             }
+            --p;
```

Comment:
What's the point of this decrement and the prior increment in `case s_headers_almost_done:`?
Could they both be removed?